src/components/Input/Input.tsx

```diff
@@ -1,22 +1,60 @@
-import React, {ChangeEvent, FC} from 'react';
-import './Input.css';
+import React, { ChangeEvent, FC } from "react";
+
+import { Skeleton } from "antd";
+import styled from "styled-components";
 
 interface InputProps {
   placeholder?: string;
   id: string;
   name: string;
   value: string;
+  isLoading?: boolean;
   onChange: (event: ChangeEvent<HTMLInputElement>) => void;
```

Comment:
лучше сделать так, чтобы `callback` принимать value: string, чтобы в каждом месте, где вы используете этот компонент, не писать `e.target.value`
```suggestion
  onChange: (event: value: string) => void;
```